### PR TITLE
Preparing for v0.1

### PR DESCRIPTION
**Pull request by ThatFrogDev**
### Added
**Commands**
- /donate
- /manage unban

**Other**
- Set up PayPal and find a way to link that to the bot for donator rewards
### Major changes
- /help - finished the command
- /manage - finished the section and make everything work properly
### Minor changes
- /math calculate - finish
- make the needed permission work at /manage ban and /manage unban

### CODE REVIEW

File: src/commands/info/about.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ module.exports = class About {
 
     client.shard.fetchClientValues("guilds.cache.size").then(results => {
       const guildCount = results.reduce((acc, guildCount) => acc + guildCount, 0);
-      const hearts = ["💖", "💝", "💓", "💗", "💘", "💟", "💕", "💞"];
+      const hearts = ["💖", "💝", "💓", "💗", "💘", "💟", "💕", "💞", "🫶"];
```
Comment (MrSerge01) [inline src/commands/info/about.js:16]: the heart is invisible for windows 10 users

File: src/commands/info/donate.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+const { EmbedBuilder, SlashCommandSubcommandBuilder } = require("discord.js");
+const { getColor } = require("../../utils/misc");
+
+module.exports = class Donate {
+    constructor() {
+    this.data = new SlashCommandSubcommandBuilder()
+        .setName("donate")
+        .setDescription("Donate money to the Nebula team.")
+    }
```
Comment (MrSerge01) [inline src/commands/info/donate.js:9]: indentation issue
```suggestion
    constructor() {
      this.data = new SlashCommandSubcommandBuilder()
        .setName("donate")
        .setDescription("Donate money to the Nebula team.");
    }
```

File: src/commands/info/donate.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        const client = interaction.client;
+
+        const embed = new EmbedBuilder()
+            .setTitle("Donate!")
+            .addFields({
+                name: "**Thanks for considering to donate!**",
+                value: [
+                    "The Nebula team will use this money for hosting the website and the bot, and paying other bills.",
+                    "We use [PayPal](https://www.google.com) for donating."
+                ].join("\n\n")},
```
Comment (MrSerge01) [inline src/commands/info/donate.js:21]: \n\n? why

File: src/commands/manage/unban.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+    const user = interaction.options.getUser("user");
+    const channel = interaction.client.channels.cache.get("979337971159420928");
+    const member = interaction.member;
+    const everyone = member.guild.roles.everyone;
+    const allMembers = await member.guild.members.fetch();
+    const allRoles = await member.guild.roles.fetch();
+    const selectedMember = allMembers.filter(m => m.user.id === user.id).get(user.id);
+    const bannedUsers = interaction.guild.bans.fetch()
+    console.log(bannedUsers)
```
Comment (MrSerge01) [inline src/commands/manage/unban.js:32]: could've removed the console.log

File: src/commands/manage/unban.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    const yes = allRoles.filter(r => r !== everyone && selectedMember._roles.includes(r.id)).sort((a, b) => Math.max([a[1].rawPosition, b[1].rawPosition]));
+
+    const embed = new EmbedBuilder()
+      .setTitle(`${selectedMember.nickname == null ? user.username : selectedMember.nickname} is unbanned.`)
+      .addFields({ name: "🔨 • Moderator", value: `${member.nickname == null ? member.user.username : member.nickname}` })
+      .setFooter({ text: `User ID: ${user.id}` })
+      .setColor(getColor(100));
+
+    if (!bannedUserArray.includes(selectedMember)) {
+      errorEmbed.setTitle("You can't unban this user, as it was never banned.")
```
Comment (MrSerge01) [inline src/commands/manage/unban.js:43]: it was => they were

File: src/game/rps.js [hunk 6]
```diff
@@ -67,19 +67,19 @@ module.exports = class Rps {
                 {
                   label: "Rock",
                   value: "rock",
-                  description: "It's about drive, it's about power",
+                  description: "It's about drive, it's about power.",
```
Comment (MrSerge01) [inline src/game/rps.js:70]: why dot

File: src/game/rps.js [hunk 7]
```diff
@@ -67,19 +67,19 @@ module.exports = class Rps {
...
                   value: "paper",
-                  description: "idk",
+                  description: "The best thing if you want to write.",
                   emoji: "📄"
                 },
                 {
                   label: "Scissors",
                   value: "scissors",
-                  description: "idk",
+                  description: "A very powerful tool, you can even cut paper with it.",
```
Comment (MrSerge01) [inline src/game/rps.js:82]: remove spoiler

File: src/commands/info/about.js (hunk 1 above)
Comment (ThatFrogDev) [reply src/commands/info/about.js:16]: wot? oh, okay lmao

File: src/commands/info/donate.js (hunk 3 above)
Comment (ThatFrogDev) [reply src/commands/info/donate.js:21]: otherwise it looks quite awful

File: src/commands/manage/unban.js (hunk 4 above)
Comment (ThatFrogDev) [reply src/commands/manage/unban.js:32]: indeed, need to do

File: src/commands/manage/unban.js (hunk 5 above)
Comment (ThatFrogDev) [reply src/commands/manage/unban.js:43]: what do you mean

File: src/game/rps.js (hunk 6 above)
Comment (ThatFrogDev) [reply src/game/rps.js:70]: idk

File: src/game/rps.js (hunk 7 above)
Comment (ThatFrogDev) [reply src/game/rps.js:82]: okay lmaoi

File: src/commands/manage/unban.js (hunk 5 above)
Comment (MrSerge01) [reply src/commands/manage/unban.js:43]: user is a person, not an object

File: src/game/rps.js (hunk 6 above)
Comment (MrSerge01) [reply src/game/rps.js:70]: remove dot

File: src/commands/manage/unban.js (hunk 5 above)
Comment (ThatFrogDev) [reply src/commands/manage/unban.js:43]: oh, u mean in text. yeah, it's changed now

File: src/game/rps.js (hunk 6 above)
Comment (ThatFrogDev) [reply src/game/rps.js:70]: ok

**Review by MrSerge01 — APPROVED** [review]
LGTM!
-The Goos.

**Pull request merged**